jupyter-config/nb-config/jupyterlab_eduhelx_submission.json

```diff
@@ -1,7 +1,7 @@
 {
   "NotebookApp": {
     "nbserver_extensions": {
-      "jupyterlab_eduhelx_submission": true
+      "eduhelx_jupyterlab_student": true
```

Comment:
You didn't rename this file or the other one in `jupyter-config/server-config/`. I assume that should be fixed?

Reply:
Yep